### ISSUE TITLE
Design Feedback

### ISSUE THREAD

**Issue by Gibson1080** [ronaldliujr/My-Personal-Project#5]
I loved how your design looks. I love that you used one navigation bar that is consistent across the entire website.I also loved that you used the same color in the entire website.The fonts however did not seem very consistent across the entire website. I would say you can work on that aspect. On your highlights page, you can increase the left margin a little bit so that its not too close to the end.I have attached a screenshot

<img width="1439" alt="Screen Shot 2022-11-11 at 2 42 45 PM" src="https://user-images.githubusercontent.com/112510165/201417792-b28722d1-7724-4689-a27d-becccb938da0.png">

**Comment by ronaldliujr** [comment]
I agree that I could definitely add a margin to the left. This would be a good change as it would eliminate some more of the empty space on the right as well.
